wiki/Gameplay/Choke/cs.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - reverse choke
+---
+
+# Choke
+
+Termín používaný, když se hráči skoro podaří udržet plné kombo když hraje [beatmapu](/wiki/Beatmap), ale kombo přeruší blízko konce mapy.
+**Obrácený choke** potom referuje na situaci, kdy hráč přeruší kombo blízko začátku mapy a následně si jej udrží až do konce.
+V [osu!](/wiki/Game_mode/osu!) a [osu!catch](/wiki/Game_mode/osu!catch) může být rozdíl mezi chokem (nebo reverse chokem) a plným kombem poměrně drastický. Je to z toho důvodu, že objekty blízko konce mají vysokou váhu kvůli akumulovanému kombu.
+
+Tento termín také používá velké množství jiných komunit, kde má zpravidla stejný, nebo podobný význam. (například [*Super Smash Bros.*](https://en.wikipedia.org/wiki/Super_Smash_Bros.) nebo [*speedrunning*](https://en.wikipedia.org/wiki/Speedrun))
+
+## Co způsobuje choke
+
+Klíčovými vlastnostmi choku jsou náhoda a nekonzistentní výkon. Přerušení komba na specifickém obtížném vzoru (paterně) může být zaviněno nedostatečnými dovednostmi hráče, nebo způsobem jakým byla beatmapa konstruována. Choke je většinou způsobený nadměrnou psychickou zátěží, nervozitou z hraní a celkovou únavou.
```

Comment:
missing a link

for ref:

`The key characteristic of a choke is its randomness and inconsistent performance. This excludes regular or pattern-specific misses or [slider breaks](/wiki/Gameplay/Judgement/Slider_break), which are usually tied to a player's skill level, or how the beatmap is constructed. Chokes are typically caused by excessive mental stress and nervousness during gameplay, as well as overall fatigue.`